src/mouse/default_gutter_handler.js

```diff
@@ -271,6 +271,9 @@ class GutterTooltip extends Tooltip {
     }
 
     hideTooltip() {
+        if(!this.$element){
```

Comment:
Could it make sense to use `this.isOpen` here instead? That way we centralize how we check whether the tooltip is currently shown

(https://github.com/ajaxorg/ace/blob/master/src/tooltip.js#L20)